doc/news/changes/minor/20231115SebastianKinnewig

```diff
@@ -0,0 +1,7 @@
+New: Introduce a new function, Utilities::Trilinos::make_rcp(),
+to create Teuchos::RCP objects, avoiding the usage of
+'operator new'.
+Fixed: Replace 'operator new' usage with function calls of
+Utilities::Trilinos::make_rcp().
```

Comment:
It wasn't really a bug, right?

Reply:
No, it was not a bug.
should I just keep that sentence in the "new" section, or should I just remove it?

Reply:
I would just remove it since the changes are not user-facing.